repo/packages/W/weavescope-probe/2/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "packagingVersion": "3.0",
+  "name": "weavescope-probe",
+  "version": "1.2.1",
+  "tags": ["microservices", "visualisation", "monitoring"],
+  "maintainer": "help@weave.works",
+  "description": "Live visualisation, monitoring and management\n\nWeave Scope produces a live interactive map of your containerized services and applications with no coding and no instrumentation",
```

Comment:
Please add a link to example at the end of description
Something like: Installation documentation: https://github.com/dcos/examples/tree/master/1.8/weave_probe

Example is basically an installation guide. Please create a PR against https://github.com/dcos/examples to add the example and please make sure to use the template from other examples. These examples finally show up in https://dcos.io/docs/1.8/usage/tutorials/